releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -2,6 +2,9 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v1.6.8 - December, 02 2019
+  * Fixed the issue of not populating content to `App Configuration name` dropdown if add the pipeline task into pipeline YAML file by Task assistant. [#202](https://github.com/Azure/AppConfiguration/issues/202)
```

Comment:
Fixed the issue that the `App Configuration name` dropdown is not populated if the pipeline task is added to the pipeline YAML file via the `assistant` UI of Tasks ([#202](https://github.com/Azure/AppConfiguration/issues/202)).

Reply:
Updated